app/eth2wrap/eth2wrap.go

```diff
@@ -74,13 +75,19 @@ func NewMultiHTTP(timeout time.Duration, addresses ...string) (Client, error) {
 	for _, address := range addresses {
 		address := address // Capture range variable.
 
+		parameters := []eth2http.Parameter{
+			eth2http.WithLogLevel(zeroLogInfo),
+			eth2http.WithAddress(address),
+			eth2http.WithTimeout(timeout),
+			eth2http.WithAllowDelayedStart(true),
+		}
+
+		if featureset.Enabled(featureset.JSONRequests) {
```

Comment:
I think you can pass the result of Enabled to WithEnforceJSON, avoiding the if statement entirely

Reply:
Indeed. Fixed.